src/test/java/api/loans/scenarios/ChangeDueDateAPITests.java

```diff
@@ -343,8 +345,8 @@ void dueDateChangeShouldClearRenewalFlagWhenSetAndNoOpenRecallsInQueue() {
     IndividualResource loanPolicy = loanPoliciesFixture.create(
       new LoanPolicyBuilder()
         .withName("loan policy")
-        .withRecallsMinimumGuaranteedLoanPeriod(org.folio.circulation.domain.policy.Period.weeks(2))
-        .rolling(org.folio.circulation.domain.policy.Period.months(1)));
+        .withRecallsMinimumGuaranteedLoanPeriod(days(3))
+        .rolling(weeks(1)));
```

Comment:
I would suggest freezing the loan date to avoid some dependencies in future

Reply:
what does freezing mean?

Reply:
fixed date for the check out

Reply:
ZonedDateTime loanDate = ZonedDateTime.of(2021, 11, 20, 13, 25, 46, 0, UTC);
    IndividualResource initialLoan = checkOutFixture.checkOutByBarcode(smallAngryPlanet, steve, loanDate);

For this solution, you don't have to change something else in the test

Reply:
fixed